subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_shadow_info

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Select the info sections that will be automatically added to the device's
+# shadow when connecting to nRF Cloud with MQTT or CoAP.
+menu "Send shadow info sections on initial connect (MQTT/CoAP)"
+
+if NRF_CLOUD_MQTT || NRF_CLOUD_COAP
+
+config NRF_CLOUD_SEND_DEVICE_STATUS
+	bool "Send device status on initial connection"
+	depends on MODEM_INFO
+	depends on MODEM_INFO_ADD_DEVICE
+	default y if NRF_CLOUD_FOTA
+	help
+	  Add "deviceInfo" section containing hardware and firmware version information.
+
+config NRF_CLOUD_SEND_DEVICE_STATUS_NETWORK
+	bool "Send network status information on initial connection"
+	depends on MODEM_INFO
+	depends on MODEM_INFO_ADD_NETWORK
+	help
+	  Add "networkInfo" section containing network connection information.
+
+config NRF_CLOUD_SEND_DEVICE_STATUS_SIM
+	bool "Send SIM card information on initial connection"
+	depends on MODEM_INFO
+	depends on MODEM_INFO_ADD_SIM
+	help
+	  Add "simInfo" section containing SIM card information.
+
+config NRF_CLOUD_SEND_DEVICE_STATUS_CONN_INF
+	bool "Send connection information on initial connection"
+	default y
+	help
+	  Add "connectionInfo" section indication how the device is connecting to nRF Cloud.
+
+config NRF_CLOUD_SEND_SERVICE_INFO_FOTA
+	bool "Send FOTA service info on initial connection"
+	help
+	  Add supported FOTA types to the "serviceInfo" section.
+	  If your application does not support certain FOTA types, this option should be disabled and your application should set this information.
```

Comment:
"should set this information" might be clearer if it said "should set one or more of the following types."

Reply:
updated, differently.